<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>app</class>
 <widget class="QMainWindow" name="app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>app</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: red;
color: white;
borders: 4px solid red;
}</string>
    </property>
    <property name="text">
     <string>POKEAPP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-app1-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/pokeball1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-app1-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/podexfinal.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pokeicon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>53</width>
      <height>56</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-app1-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/001.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pokename">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>90</y>
      <width>68</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: black;
color: white
;
}</string>
    </property>
    <property name="text">
     <string>BULBASUR 001</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

  color: white;

  background: #e30052 ;
  border: 3px solid #e30052;
  border-radius: 20px;
  box-shadow: 0 5px 20px 3px rgba(0, 0, 0, 0.6);
}
</string>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Next">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

  color: white;

  background: #e30052 ;
  border: 3px solid #e30052;
  border-radius: 20px;
  box-shadow: 0 5px 20px 3px rgba(0, 0, 0, 0.6);
}
</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Description">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
 {

  color: white;

  background: #e30052 ;
  border: 3px solid #e30052;
  border-radius: 20px;
  box-shadow: 0 5px 20px 3px rgba(0, 0, 0, 0.6);
}
</string>
    </property>
    <property name="text">
     <string>DESCRIPTION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_descrip">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>58</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: white;
border-style: none;

}</string>
    </property>
    <property name="text">
     <string>Name: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Type">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>74</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: white;
border-style: none;

}</string>
    </property>
    <property name="text">
     <string>Tipos:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="HP">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>91</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: white;
border-style: none;

}</string>
    </property>
    <property name="text">
     <string>HP:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Attack">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: white;
border-style: none;

}</string>
    </property>
    <property name="text">
     <string>Attack:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Defense">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: white;
border-style: none;

}</string>
    </property>
    <property name="text">
     <string>Defense:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="SpAttack">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: white;
border-style: none;

}</string>
    </property>
    <property name="text">
     <string>Sp.Attack:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="SpDefense">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: white;
border-style: none;

}</string>
    </property>
    <property name="text">
     <string>Sp.Defense:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Speed">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: white;
border-style: none;

}</string>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Number">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>73</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: white;
border-style: none;

}</string>
    </property>
    <property name="text">
     <string>Number:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
